app/views/sections/_form.html.erb

```diff
@@ -1,79 +1,141 @@
-<div class='row'>
-  <div class="col-md-8">
+<% content_for :head do %>
+  <%= javascript_include_tag "vendor/jquery-1.11.0.min" %>
```

Comment:
Is this to make up for the fact that the design system doesn't typically bundle jquery?